docs/user-guide/install-zowe-server-install-wizard.md

```diff
@@ -0,0 +1,242 @@
+|STC|**Purpose**:<br /> ZWESLSTC is the job for running Zowe's webservers. ZWESISTC runs the APF authorized cross-memory server. The ZWESASTC job is started by ZWESISTC on an as-needed basis.<br /><br />**Action**:<br /> Copy the members ZWESLSTC, ZWESISTC, and ZWESASTC into your desired PROCLIB. If the job names are customized, modify the job name YAML values in `zowe.setup.security.stcs`|[ZWEISTC](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEISTC)|
+|(Optional) VSAM for Caching Service (NYI)|**Purpose:**<br /> To use VSAM as your storage method forteh Caching Service. For more information, see [Using VSAM as a storage solution through the Caching service](../extend/extend-apiml/api-mediation-vsam.md).<br /><br />**Action:**<br /> Create a RLM or NONRLM dataset for the Caching service, and set the name into the YAML value `components.caching-service.storage.vsam.name`|[ZWECSVSM](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWECSVSM)|
+
+
+## Final Review
+
+After completing the steps presented in the Wizard, a summary is provided indicating which steps were completed, skipped, or have errors. Errors are stored in the View Job Output. You can revisit any step to retry performing the step. You also have the option to export the final generated YAML file.
+
+## Troubleshooting
+
+### Failure to establish a TLS connection
+
+When attempting to establish a TLS connection, you may encounter the following message:
+
+```
+Client network socket disconnected before secure TLS connection was established
+```
+
+If you receive this message, go back to the Connection page and attempt to re-establish the connection. If the connection cannot be established, restart the Wizard.
+
+### Unable to continue with Wizard installation
+
+If you encounter strange behavior that prohibits you from continuing with wizard installation, we recommend you follow this procedure:
+
+1. View the Job output within the wizard.
+2. If the error is not clear from the Job output, view the output of the log file according to your platform:
+
+    <details>
+    <summary>For Windows</summary>
+
+    `%USERPROFILE%\AppData\Roaming{app name}\logs\main.log`
+    </details>
+    
+    <details>
+    <summary>For macOS</summary>
+
+    ` ~/Library/Logs/{app name}/main.log`
+    </details>
+
+    <details>
+    <summary>For Linux</summary>
+    
+    ` ~/.config/{app name}/logs/main.log`
+    </details>
+
+If you are still unsure how to proceed, make a backup of these log files, and then use the following procedure to remove the Wizard's cache. 
```

Comment:
"If you are still unsure how to proceed, make a backup of these log files, and then use the following procedure to remove the Wizard's cache." -->

"If you are still unsure how to proceed, (optional) make a backup of these log files, and then use the following procedure to remove the Wizard's cache."